macos/FluentUI/Colors.swift

```diff
@@ -0,0 +1,304 @@
+			case .warningPrimary:
+				return "warningPrimary"
+			case .warningTint40:
+				return "warningTint40"
+			case .warningTint30:
+				return "warningTint30"
+			case .warningTint20:
+				return "warningTint20"
+			case .warningTint10:
+				return "warningTint10"
+			case .warningShade30:
+				return "warningShade30"
+			case .warningShade20:
+				return "warningShade20"
+			case .warningShade10:
+				return "warningShade10"
+			case .successPrimary:
+				return "successPrimary"
+			case .successTint40:
+				return "successTint40"
+			case .successTint30:
+				return "successTint30"
+			case .successTint20:
+				return "successTint20"
+			case .successTint10:
+				return "successTint10"
+			case .successShade30:
+				return "successShade30"
+			case .successShade20:
+				return "successShade20"
+			case .successShade10:
+				return "successShade10"
+			case .presenceAvailable:
+				return "presenceAvailable"
+			case .presenceAway:
+				return "presenceAway"
+			case .presenceBlocked:
+				return "presenceBlocked"
+			case .presenceBusy:
+				return "presenceBusy"
+			case .presenceDnd:
+				return "presenceOffline"
+			case .presenceOffline:
+				return "presenceOffline"
+			case .presenceOof:
+				return "presenceOof"
+			case .presenceUnknown:
+				return "presenceUnknown"
+			}
+		}
+	}
+	
+	// MARK: Physical - grays
+	
+	@objc public static let gray950: NSColor = Palette.gray950.color
+	@objc public static let gray900: NSColor = Palette.gray900.color
+	@objc public static let gray800: NSColor = Palette.gray800.color
+	@objc public static let gray700: NSColor = Palette.gray700.color
+	@objc public static let gray600: NSColor = Palette.gray600.color
+	@objc public static let gray500: NSColor = Palette.gray500.color
+	@objc public static let gray400: NSColor = Palette.gray400.color
+	@objc public static let gray300: NSColor = Palette.gray300.color
+	@objc public static let gray200: NSColor = Palette.gray200.color
+	@objc public static let gray100: NSColor = Palette.gray100.color
+	@objc public static let gray50: NSColor = Palette.gray50.color
+	@objc public static let gray25: NSColor = Palette.gray25.color
+	
+	// MARK: Physical - Non-grays
+	
+	@objc public static let error: NSColor = Palette.dangerPrimary.color
+	@objc public static let warning: NSColor = Palette.warningPrimary.color
+	
+	@objc public static var avatarBackgroundColors: [NSColor] = [
+		Palette.cyanBlue10.color,
+		Palette.red10.color,
+		Palette.magenta20.color,
+		Palette.green10.color,
+		Palette.magentaPink10.color,
+		Palette.cyanBlue20.color,
+		Palette.orange20.color,
+		Palette.cyan20.color,
+		Palette.orangeYellow20.color,
+		Palette.red20.color,
+		Palette.blue10.color,
+		Palette.magenta10.color,
+		Palette.gray40.color,
+		Palette.green20.color,
+		Palette.blueMagenta20.color,
+		Palette.pinkRed10.color,
+		Palette.gray30.color,
+		Palette.blueMagenta30.color,
+		Palette.gray20.color,
+		Palette.cyan30.color,
+		Palette.orange30.color
+	]
```

Comment:
shouldn't this be in AvatarView instead?

Reply:
Agreed, though I'm just gonna remove these for now until they're needed.

Reply:
it should be needed for Mac AvatarView ....

Reply:
I don't see it being used anywhere in the code...

Reply:
static let avatarBackgroundColors: [NSColor] = [] in AvatarView.swit should be replace with this new list so that mac and iOS have the same avatar colors.